api/src/geolocate/get.handler.ts

```diff
@@ -21,24 +20,48 @@ async function getGeoDataForLatLong(
   params: SqlParametersList,
   geoid: string,
   table: string,
-): Promise<string | undefined> {
+): Promise<BoundedGeoDatum | undefined> {
+  // TODO(breuch): Consider updating this to all geoids when we support
+  // county, states, etc.
   const executeParams: ExecuteStatementRequest = {
     database: process.env.DATABASE_NAME ?? 'postgres',
     resourceArn: process.env.RESOURCE_ARN ?? '',
     schema: SCHEMA,
     secretArn: process.env.CREDENTIALS_SECRET ?? '',
-    sql: `SELECT ${geoid}
-          FROM ${table}
-          WHERE geom && ST_SetSRID(ST_Point(:long, :lat), 4326)
-          LIMIT 1`,
+    sql: `
+      WITH target_row AS (
+        SELECT ${geoid} AS id, geom AS geom
+        FROM ${table}
+        WHERE geom && ST_SetSRID(ST_Point(:long, :lat), 4326)
+          ORDER BY id ASC
+          LIMIT 1
+      )
+      SELECT
+        STRING_AGG(id, '|') AS id,
+        ST_EXTENT(geom) AS bounding_box
+      FROM target_row`,
     parameters: params,
   };
+  //Example response: NY0700789	'BOX(-74.258843 40.476578,-73.700169 40.917705)'
   const results = await rdsService.executeStatement(executeParams).promise();
-  const geoids = results.records?.map((record: FieldList) => record[0].stringValue) ?? [];
+  const geoID = results.records ? (results.records[0] || undefined) : undefined;
+  if(!geoID) return undefined;
 
-  // TODO(breuch): Consider updating this to all geoids when we support
-  // county, states, etc.
-  return geoids[0];
+  const [rawId, rawBoundingBox] = geoID;
+
+  const boundingBoxVals = rawBoundingBox.stringValue
+  ?.replace('BOX(', '').replace(')', '') // strip out the BOX() wrapper
```

Comment:
these comments are great, but can we also just leave one above this block like 'Query returns bounding box in the format '...'.' And link the postgis documentation?

Reply:
Done. I made it a bit of an essay but I figure clarity is better.